src/main/scala/it/pagopa/interop/backendforfrontend/api/impl/PrivacyNoticesApiServiceImpl.scala

```diff
@@ -84,11 +85,11 @@ final case class PrivacyNoticesApiServiceImpl(
         .unlessA(latest.privacyNoticeVersion.versionId == seed.latestVersionId)
       _        <- privacyNoticesService.put(
         PersistentModel.UserPrivacyNotice(
-          pk = s"${PersistentModel.UserPrivacyNotice.pkPrefix}$pnUuid",
-          sk = s"${PersistentModel.UserPrivacyNotice.skPrefix}$userUuid#${latest.privacyNoticeVersion.version}",
-          pnId = pnUuid,
+          pnIdWithUserId = s"$pnUuid#$userUuid",
+          versionNumber = latest.privacyNoticeVersion.version,
+          privacyNoticeId = pnUuid,
           userId = userUuid,
-          acceptedAt = OffsetDateTimeSupplier.get(),
+          acceptedAt = OffsetDateTimeSupplier.get().truncatedTo(ChronoUnit.SECONDS),
```

Comment:
[same](https://github.com/pagopa/interop-be-jobs/pull/114#discussion_r1213064052)

Reply:
The attribute acceptedAt is the time of the operation and I supposed that up to the seconds was enough. If you want I remove it

Reply:
Fixed